system/bluetooth_monitor/service/l2ping.cpp

```diff
@@ -0,0 +1,364 @@
+// Copyright 2022 Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// This code is derived from BlueZ with following copyrights.
+
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Is GPL-2.0 License allowed in autoware?

Reply:
Actually, considering the maintenance, I believe no. :pleading_face: 
We should prepare another repository (and add it `autoware.repos`), or change the code not to use licenses that aren't compatible with Apache-2.0.

@mitsudome-r @xmfcx @esteve Any thoughts?

Reply:
@wep21 Thank you for the comment. I will fix the code so as not to violate the GPL licence, considering command line execution in another process.

Reply:
@kenji-miyake if this is isolated enough, it could be fine to accept it as long as it does not touch the rest of the code (e.g. links to it), but I think it's best to avoid it if there are other options.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/862/commits/ad51b04730ea415375648c962fd3c51987c82ee7.
We no longer use any codes derived from bluez.